docs/api/general_rest_usage.md

```diff
@@ -208,6 +208,17 @@ The REST API supports two authentication methods:
 
 Session-based is the default authentication method, as it is needed for UI.
 
+!!! note
+    
+    There are some API endpoints that might be accessible for the Anonymous User outside the Back Office due to missing permissions or limitations. To prevent that, you can rely on the Symfony securing URL patterns mechanizm called [access_control](https://symfony.com/doc/3.4/security/access_control.html). The example below shows how to disallow listing Content Types for the not-authenticated user.
```

Comment:
```suggestion
    There are some API endpoints that might be accessible to the Anonymous User outside of the Back Office due to missing permissions or limitations.
    To prevent that, you can rely on the Symfony securing URL patterns mechanism called [access_control](https://symfony.com/doc/3.4/security/access_control.html).
    The example below shows you how to block listing Content Types for the non-authenticated users.
```